msal/build.gradle

```diff
@@ -188,12 +188,12 @@ task sourcesJar(type: Jar) {
 
 // In dev, we want to keep the dependencies (common4j, common) to 1.0.+ to be able to be consumed by daily dev pipeline.
 // In release/*, we change these to specific versions being consumed.
-def commonVersion = "1.0.+"
+def commonVersion = "17.6.0-RC1"
 if (project.hasProperty("distCommonVersion")) {
     commonVersion = project.distCommonVersion
 }
 // Used for testfixtures
-def common4jVersion = "1.0.+"
+def common4jVersion = "14.6.0-RC1"
```

Comment:
I don't think this is right either? Neither the above.

Reply:
That might because I merge release branch into the test app branch in order to have both test environment. I will check the content in the dev branch and revert it back.